pyleros/types.py

```diff
@@ -0,0 +1,10 @@
+from myhdl import enum
+
+t_decSignal = enum('op', 'al_ena', 'ah_ena', 'log_add', 'add_sub', \
+					'shr', 'sel_imm', 'store', 'outp', 'inp', 'br_op', 'jal', \
+					'loadh', 'indls')
+
+alu_op_type = enum('LD', 'AND', 'OR', 'XOR')
+
```

Comment:
Need to be consistent with naming, in one case you use `t_` and in another you use `_type` also mix `_` separator and CamelCase.  I prefer the second format, also the second format it is clear what is being enumerated, in the first case I am not sure what it is by the name given.